### ISSUE TITLE
Please provide me an option or code for disable scrolling on fixed screen data

### ISSUE THREAD

**Issue by waqaryounusjatt788** [syncfusion/flutter-widgets#53]
Hey, 
Please see in the below linked vide, I have set fixed screen data but its screen only data cell, kindly advice me how to stop this scrolling its impact on my whole Screen  where i have set multiple widget  of scrolling.

Just like we were did in ListView  physics = NeverScrollableScrollPhysics() for disable scrolling, 
Please provide me an option or code for disable scrolling on fixed screen data
![sample](https://user-images.githubusercontent.com/65191117/98940790-52ddc400-250d-11eb-80d5-5827f8b9c1e3.gif)

**Comment by elliot111** [comment]
I have the same problem, please solve it as soon as possible, thank you

**Comment by NeelakandanKannan** [comment]
Hi @waqaryounusjatt788 , @elliot111 ,

We have considered this request. We will provide the support to set the physics for horizontal and vertical scrollbar in our upcoming weekly pub release which is scheduled on 24th November 2020. Please refer the below feedback link,

https://www.syncfusion.com/feedback/19678/support-to-show-scrollbars-always-and-set-the-physics-for-both-the-scrollbars

**Comment by waqaryounusjatt788** [comment]
@elliot111  just use IgnorePointer to stop these scrolling

` IgnorePointer(
            ignoring: true,
              child: SfDataGrid()
)`

**Comment by elliot111** [comment]
thanks，I will try .



---Original---

**Comment by NeelakandanKannan** [comment]
Hi @waqaryounusjatt788 / @elliot111 ,

Unfortunately if you are using IgnorePointer widget, you won't be able to sort the columns in case you want sorting feature. So, as promised we have provided the support to disable the scrolling. It can be done by setting the [NeverScrollableScrollPhysics](https://api.flutter.dev/flutter/widgets/NeverScrollableScrollPhysics-class.html) to the below properties,

[horizontalScrollPhysics](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/SfDataGrid/horizontalScrollPhysics.html) - To disable the scrolling for horizontal scroll bar.

[verticalScrollPhysics](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/SfDataGrid/verticalScrollPhysics.html) - To disable the scrolling for vertical scrollbar.

Example code,
```dart
                child: SfDataGrid(
                  source: _employeeDataGridSource,
                  columns: getColumns(),
                  gridLinesVisibility: GridLinesVisibility.both,
                  verticalScrollPhysics: NeverScrollableScrollPhysics(),
                  horizontalScrollPhysics: NeverScrollableScrollPhysics(),
                ),
```

**Comment by elliot111** [comment]
Hi,
The SfCalendar doesn't seem to have this function,&nbsp;I want The SfCalendar&nbsp;to stop sliding horizontally. What should I do?It doesn't seem to have this property




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "syncfusion/flutter-widgets"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年11月26日(星期四) 下午2:06
收件人:&nbsp;"syncfusion/flutter-widgets"<flutter-widgets@noreply.github.com&gt;;
抄送:&nbsp;"徐龙洋"<568610207@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [syncfusion/flutter-widgets] Please provide me an option or code for disable scrolling on fixed screen data (#53)





 
Hi @waqaryounusjatt788 / @elliot111 ,
 
Unfortunately if you are using IgnorePointer widget, you won't be able to sort the columns in case you want sorting feature. So, as promised we have provided the support to disable the scrolling. It can be done by setting the NeverScrollableScrollPhysics to the below properties,
 
horizontalScrollPhysics - To disable the scrolling for horizontal scroll bar.
 
verticalScrollPhysics - To disable the scrolling for vertical scrollbar.
 
Example code,
                 child: SfDataGrid(                   source: _employeeDataGridSource,                   columns: getColumns(),                   gridLinesVisibility: GridLinesVisibility.both,                   verticalScrollPhysics: NeverScrollableScrollPhysics(),                   horizontalScrollPhysics: NeverScrollableScrollPhysics(),                 ),

**Comment by Indumathi1195R** [comment]
Hi @elliot111 ,

Based on the provided information, we have checked your requirement “**Stop sliding horizontally in Flutter event calendar**” and it can be achieved by using the property of _navigationDirection_ in MonthViewSettings. By default, the value of the navigationDirection is horizontal. Kindly use the navigationDirection values as **MonthNavigationDirection.vertical** to restrict the horizontal scrolling. 

Please find the code snippet for the same.

**Code snippet:**
``
child: SfCalendar(
  view: CalendarView.month,
  monthViewSettings: MonthViewSettings(
      navigationDirection: MonthNavigationDirection.vertical),
),

``

Also, please find the UG link for the same.

**UG link:**
https://help.syncfusion.com/flutter/calendar/month-view#month-navigation-direction

We hope that the above helps you. If it is not your requirement, please share more details about your requirement with clearly. It will be helpful for us to analyze and provide you a solution at the earliest.

Regards,
Indumathi R

**Comment by elliot111** [comment]
Hi,Thank you for your reply,It resolves the sliding issue in the month view, but horizontal sliding cannot be disabled in the week view.






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "syncfusion/flutter-widgets"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年12月1日(星期二) 下午4:37
收件人:&nbsp;"syncfusion/flutter-widgets"<flutter-widgets@noreply.github.com&gt;;
抄送:&nbsp;"徐龙洋"<568610207@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [syncfusion/flutter-widgets] Please provide me an option or code for disable scrolling on fixed screen data (#53)





 
Hi @elliot111 ,
 
Based on the provided information, we have checked your requirement “Stop sliding horizontally in Flutter event calendar” and it can be achieved by using the property of navigationDirection in MonthViewSettings. By default, the value of the navigationDirection is horizontal. Kindly use the navigationDirection values as MonthNavigationDirection.vertical to restrict the horizontal scrolling.
 
Please find the code snippet for the same.
 
Code snippet:
 ``
 child: SfCalendar(
 view: CalendarView.month,
 monthViewSettings: MonthViewSettings(
 navigationDirection: MonthNavigationDirection.vertical),
 ),
 
``
 
Also, please find the UG link for the same.
 
UG link:
 https://help.syncfusion.com/flutter/calendar/month-view#month-navigation-direction
 
We hope that the above helps you. If it is not your requirement, please share more details about your requirement with clearly. It will be helpful for us to analyze and provide you a solution at the earliest.
 
Regards,
 Indumathi R

**Comment by Indumathi1195R** [comment]
Hi @elliot111,

We have logged the feature request for “**Providing a support for enable/disable swiping interaction in the Flutter event calendar (SfCalendar)**” in the Flutter event calendar. We will implement this feature in any of our upcoming releases. 

At the planning stage for every release cycle, we review all open features and identify features for implementation based on specific parameters including product vision, technological feasibility, and customer interest. We will let you know when this feature is implemented. We appreciate your patience until then.

Thank you for requesting this feature and helping us define it. We are always trying to make our products better and feature requests like yours are a key part of our product growth efforts.
 
You can also communicate with us regarding the open features at any time using our Feature Report page. 
 
**Feedback link:** https://www.syncfusion.com/feedback/20195/providing-a-support-for-enable-disable-swiping-interaction-in-the-flutter-event

If you have any more specifications/suggestions to the feature request, you can add it as a comment in the portal and cast your vote to make it count.

Regards,
Indumathi R
